src/plays/code-editor/styles.css

```diff
@@ -0,0 +1,66 @@
+/* enter stlyes here */
```

Comment:
Can you please prefix a unique class name to all the styles?

Reply:
Okay

Reply:
Do you want me to change all the class names or a specific one ...?

Reply:
All, what I meant is, add a unique class prefix like,

.code-editor .existing-class

Reply:
Okay